learn/on-premise/azure/install/initilize-setup.md

```diff
@@ -19,3 +19,13 @@ sidebar_label: "Initialize Setup"
 - Go to  <http://Platform-Instance-IP:8080> as directed by the above command output to do the WME setup.
 
     [![setup initialization](/learn/assets/wme-setup/wavemaker-setup-initialization.jpg)](/learn/assets/wme-setup/wavemaker-setup-initialization.jpg)
+
+- From WME version 10.5.1 version users can use their own user for communication between Platform, Studio Workspace and Appdeployment Instances.for use custom user for communication and operations do [prerequisites](../install-prerequisites.md) as per given steps.
```

Comment:
Move this line to before CW access URL line.
Change like this.
WaveMaker Platform needs users for ssh access between instances(Platform and Studioworkspace Instance, AppDeployment Instance).  You can provide your own user to provide fine grained access or Platform will create user call wavemaker with it auto generated keys.
If you want to provide your own user, select yes and then give username and path to ssh key. 
With this user and key, Platform should able to make ssh connection to StudioWorkspace Instance and App Deployment Instance.  
Platform will add below permissions to the given user
     1)
     2)
     3)